proxies/venue-graphql-proxy/src/schema/resolvers.ts

```diff
@@ -0,0 +1,15 @@
+import Venue from '../resolvers/Venue';
+import venueQueryResolver from '../resolvers/venueQueryResolver';
+import venuesByIdsResolver from '../resolvers/venuesByIdsResolver';
+
+const resolvers = {
+  Query: {
+    venue: venueQueryResolver,
+    venuesByIds: venuesByIdsResolver,
+    // events: eventsQueryResolver,
+  },
+  Venue,
+  // EventsConnection: LinkedPaginatedConnectionResolver,
+  // Event,
+};
```

Comment:
NOTE: the eventsQueryResolver and the Event type is already commented out!

Reply:
Can remove the comments as well